### PR TITLE
Update response for schedule list

### PR DESCRIPTION
**Pull request by imnutz**
Add `executing_user_id` and `description` to the response

### CODE REVIEW

**Review by tung-vu-td — APPROVED** [review]
LGTM

File: src/main/java/com/treasuredata/client/model/TDSavedQuery.java [hunk 1]
```diff
@@ -181,6 +187,16 @@ public TDJob.EngineVersion getEngineVersion()
...
+        return executingUserId;
+    }
+
+    public String getDescription()
+    {
+        return description;
+    }
+
     @Override
     public String toString()
```
Comment (yuokada) [inline src/main/java/com/treasuredata/client/model/TDSavedQuery.java:201]: It's better to update the `toString` method, too.

File: src/main/java/com/treasuredata/client/model/TDSavedQuery.java [hunk 2]
```diff
@@ -198,6 +214,8 @@ public String toString()
                 ", result='" + result + '\'' +
                 ", nextTime='" + nextTime + '\'' +
                 ", engineVersion='" + engineVersion + '\'' +
+                ", executingUserid='" + executingUserId + '\'' +
```
Comment (yuokada) [inline src/main/java/com/treasuredata/client/model/TDSavedQuery.java:217]:
```suggestion
                ", engineVersion=" + engineVersion +
                ", executingUserId=" + executingUserId +
```

NIT: Single quotes aren't necessary.

**Review by yuokada — APPROVED** [review]

**Review by ylzen — APPROVED** [review]
Reviewed for Data Exfiltrations.

Please address existing review comments.

**Pull request merged**